### PR TITLE
Add quality check for unresolved merge conflicts

### LINKED ISSUE #4633: Add merge conflicts check to quality spec
I checked in some merge conflicts inside .ronn files by accident :joy: it would be awesome if the quality spec failed when that happens.

### PR DESCRIPTION
**Pull request by RochesterinNYC**
- Closes #4633

### CODE REVIEW

File: spec/quality_spec.rb [hunk 1]
```diff
@@ -115,19 +131,29 @@ def check_for_specific_pronouns(filename)
   end
 
   it "does not include any leftover debugging or development mechanisms" do
-    included = /spec/
-    exempt = /quality\_spec\.rb/
+    exempt = %r{quality_spec.rb|support/helpers}
```
Comment (segiddins) [inline spec/quality_spec.rb]: Why this change?

**Comment by segiddins** [comment]
👍🏻

File: spec/quality_spec.rb (hunk 1 above)
Comment (RochesterinNYC) [inline spec/quality_spec.rb]: If you're referring to the reflex change to `%r`, it was to appease Rubocop. Not 100% sure why usage of `|` triggers Rubocop to suggest using `%r`, though I suspect it's actually the usage of `/` that triggers it.

**Comment by segiddins** [comment]
@homu r+
